src/containers/pages/JurisdictionAssignment/JurisdictionTable/index.tsx

```diff
@@ -87,13 +82,17 @@ export interface JurisdictionSelectorTableProps {
   fetchPlanCreator: ActionCreator<AddPlanDefinitionAction>;
   selectedLeafNodes: TreeNode[];
   leafNodes: TreeNode[];
+  autoSelectionFlow: boolean;
+  deselectAllNodesCreator: typeof deselectAllNodes;
 }
 
 const defaultProps = {
   autoSelectNodesCreator: autoSelectNodes,
+  autoSelectionFlow: false,
   currentChildren: [],
   currentParentId: undefined,
   currentParentNode: undefined,
```

Comment:
The fields that have default undefined could be handled in the interface e.g. `currentParentId?` instead of what you have here